projects/kit/components/input-phone/input-phone.component.ts

```diff
@@ -156,6 +156,14 @@ export class TuiInputPhoneComponent
             : this.search || '';
     }
 
+    set nativeValue(value: string) {
+        setTimeout(() => {
```

Comment:
@imitrakov  Why do you need the setTimeout here?

Reply:
Or return as I did before, when check  countryCode in pipe textMask? :)

Reply:
I see now. Previously active zone worked a little different and by the time pipe kicked in `focused` was already `false`. Looks like it's not as easy as I though 🤔